loadrunner_professional/README.md

```diff
@@ -0,0 +1,58 @@
+# LoadRunner Professional
+
+## Overview
+
+OpenText™ LoadRunner Professional provides a load testing solution that enables you to test the performance of diverse application types, and to identify and resolve issues before applications go live.
+
+LoadRunner Controller is a tool for creating and controlling LoadRunner Professional scenarios. A scenario defines the events that occur during each testing session. It controls the number of users to emulate (virtual users, or Vusers), the actions they perform, and the machines on which they run their emulations. You use scenarios to create load tests to check the reliability and strength of your servers.
+
+You can configure Controller to push real-time metrics and data from scenario runs to Datadog.
+
+
+## Setup
+
+Configure LoadRunner Controller to push data to Datadog. You can choose whether to send scenario information and/or run metrics. In Datadog, you can install a dashboard to view the data in pre-configured widgets.
+
+1.	In the Controller toolbar, select __Tools > Datadog Configuration__.
```

Comment:
Since users will be reading this from the Datadog app (or Datadog docs) it would be useful to add another first step here to navigate to the LoadRunner platform (and a link).

Reply:
LoadRunner Controller is a desktop application, maybe a first step saying something like "open Controller"?

Reply:
Yes that would be helpful!